grafana-aws-metrics/main.go

```diff
@@ -0,0 +1,201 @@
+package main
+
+import (
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/aws/aws-sdk-go/service/sts/stsiface"
+	"github.com/pkg/errors"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	vpcID = "VPC_ID"
+)
+
+type clientConfig struct {
+	Client      *clientcmdapi.Config
+	ClusterName string
+	ContextName string
+	roleARN     string
+	sts         stsiface.STSAPI
+}
+
+func main() {
+	lambda.Start(handler)
+}
+
+func handler() {
+	log.Info("Getting existing ELB limits")
+	err := getSetELBLimits()
+	if err != nil {
+		log.WithError(err).Fatal("Unable to get the existing ELB limits and set the CloudWatch metric data")
```

Comment:
Instead of calling 'Fatal` to stop the Lambda function, do we just want to call `Error` instead and try to get the other resources? Just a thought.